### PR TITLE
Develop

### LINKED ISSUE #3: 3 - Document Branching strategy for Giflow and add to the docs
As a developer I need to understand the branching strategy which includes

Creating Feature branch 

- project
- hotfixes

Merging strategy for feature -> develop -> release -> main

### LINKED ISSUE #5: 1 - Add the definition of done to the docs folder
The follow outlines the Definition of Done for the team

- Code can be deployed i.e is without errors
- Peer reviewed by at least one team member
- Where testing is required that it is completed and the results are attached
- Acceptance criteria met for each issue

### LINKED ISSUE #6: 1- Add the Definition of Ready to the docs folder
Add the following definition of Ready to the GitHub Documentation Branch

- The user story is clear
- Where applicable it is testable and feasible
- Well defined
- User Acceptance criteria is defined
- Dependencies are identified

### LINKED ISSUE #20: 3 - Research About Container Registry/Repository to store Docker Builds
As a Team member of CleanerCode project I have to research available container registries or repositories so that our team can select the best option available.

### PR DESCRIPTION
**Pull request by JoseSunoj**
Please have a look at the proposed security tools

### CODE REVIEW

**Review by L00162876 — APPROVED** [review]

File: docs/definitionOfDone.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Definition of Done
```
Comment (sharonO13) [inline docs/definitionOfDone.md:1]: Should the Definition of Done not be what was agreed upon and listed in the Github Issue? There seems to be extra items here which we will have to discuss as team before we add them in

File: docs/definitionOfReady.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Definition of Ready
```
Comment (sharonO13) [inline docs/definitionOfReady.md:1]: The same applies here, should the Definition of Ready not be the list of items we agreed as a team from the Github issues, if there are any additional criteria we will need to discuss as a team.

**Review by sharonO13 — CHANGES_REQUESTED** [review]
Great work on the Gitflow documentation. 
We did agree however to have a documentation branch to manage documentation only branching and merging from too and from that branch, while the code would be what is managed in the Gitflow process.

File: docs/definitionOfDone.md (hunk 1 above)
Comment (JoseSunoj) [reply docs/definitionOfDone.md:1]: If we strictly follow the GitFlow workflow, then there must be two main branches, (main, develop). All others are supporting branches, I referred to the link for the original proposer and also the article link I posted on slack. 

I have no objection to edit the files in question.

File: docs/definitionOfReady.md (hunk 2 above)
Comment (JoseSunoj) [reply docs/definitionOfReady.md:1]: Please write what are the items you have an objection about.

**Pull request merged**